tests/test_csrf.py

```diff
@@ -0,0 +1,7 @@
+def test_csrf_files(flask_test_client):
```

Comment:
This test name doesn't make sense to me - what does the "files" bit represent?

Reply:
I will change the name to more specific

Reply:
Got rid of this test file for this sprint. I will try to write a test on later sprints. OWASP zap scan added to .github